tgstation.dme

```diff
@@ -9343,6 +9343,7 @@
 #include "modular_zubbers\code\modules\zantag_tickets\preferences.dm"
 #include "modular_zubbers\code\modules\~donator\choicebeacon.dm"
 #include "modular_zubbers\code\modules\~donator\mothdonator.dm"
+#include "modular_zubbers\code\modules\species_synthesizer\sing_tones.dm"
```

Comment:
Don't manually include the include file, you need to let byond tick it by either using the vscode's extension ticking or DM editor's.